src/DataFramesMeta.jl

```diff
@@ -32,7 +32,7 @@ replace_syms!(q::QuoteNode, membernames) =
 replace_syms!(e::Expr, membernames) =
     if onearg(e, :^)
         e.args[2]
-    elseif onearg(e, :_I_)
+    elseif onearg(e, :cols)
```

Comment:
I propose to leave `_I_` and print deprecation warning that it will be removed in the future, but keep new `cols` (so essentially we have another `elseif` for time being).